bin/csv_to_krun_json

```diff
@@ -63,35 +32,21 @@ def create_cli_parser():
                    'SMP Tue Sep 6 15:42:33 UTC 2016 x86_64 x86_64 x86_64 '
                    'GNU/Linux" results.csv\n'
                    % script)
-    parser = argparse.ArgumentParser(description)
-    parser.add_argument('csv_files',
-                        nargs='+',
-                        action='append',
-                        default=[],
-                        type=str,
-                        help='One or more CSV result files.')
-    parser.add_argument('--language', '-l',
-                        dest='language',
-                        action='store',
-                        default='java-default',
-                        type=str,
+    parser = argparse.ArgumentParser(description,
+                                     formatter_class=argparse.RawTextHelpFormatter)
+    parser.add_argument('csv_files', nargs='+', action='append', default=[],
+                        type=str, help='One or more CSV result files.')
+    parser.add_argument('--language', '-l', dest='language', action='store',
+                        default='java-default', type=str,
```

Comment:
I don't think people will necessarily pick up the lowercase/titlecase thing, so it's probably best to spell it out and remove the defaults.